modules/modules.nix

```diff
@@ -157,6 +157,7 @@ let
     (loadModule ./services/udiskie.nix { })
     (loadModule ./services/unclutter.nix { })
     (loadModule ./services/unison.nix { condition = hostPlatform.isLinux; })
+    (loadModule ./services/waybar.nix { })
```

Comment:
Does waybar support macOS? If not then please mark it as Linux only like for unison above.